src/main/java/Duke.java

```diff
@@ -24,13 +27,18 @@ public void run() {
         sayGreeting();
```

Comment:
Think it's great that you separate the welcome greeting and goodbye into functions, to get them 'out of the way'!